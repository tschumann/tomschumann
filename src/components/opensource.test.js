import React from 'react';
import renderer from 'react-test-renderer';
import OpenSource from './opensource';

test('test component renders', () => {
	const component = renderer.create(<OpenSource title="Project" link="http://git.lan/project" key="1"/>);
	const tree = component.toJSON();

	expect(tree).toMatchSnapshot();
});

