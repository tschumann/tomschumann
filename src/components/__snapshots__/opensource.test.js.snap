// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component renders 1`] = `
<div>
  <strong>
    Project
  </strong>
  : 
  <a
    href="http://git.lan/project"
    target="_blank"
  >
    http://git.lan/project
  </a>
</div>
`;
